org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -10,10 +10,20 @@
     <method name="turnOff"/>
     
     <!-- Get the current ambient temperature -->
-    <method name="getCurrentTemperature">
+    <method name="getAmbientTemperature">
```

Comment:
That shouldn't be a problem, but ideally your schema should be merged first before we can make that change. Also, in the Wink Thermostat, since Wink only accepts Celsius, we would need to add a helper method called `converter`